runelite-client/src/main/java/net/runelite/client/plugins/deathindicator/DeathIndicatorConfig.java

```diff
@@ -0,0 +1,46 @@
+package net.runelite.client.plugins.deathindicator;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+		keyName = "deathindicator",
+		name = "Death Indicator",
+		description = "Configuration for the death indicator plugin"
```

Comment:
Indent should be fixed, these should only be 1 tab

Reply:
noted on the indents, thanks, will fix.

Reply:
fixed indents, thanks for the review :)